### PR TITLE
CompletedJS Forms. Mohamed Elmessiry

### PR DESCRIPTION
**Pull request by Mohamed-Elmessiry**
Screen Shot 2021-02-03 at 4 12 04 PM
Screen Shot 2021-02-03 at 4 11 33 PM
Screen Shot 2021-02-03 at 4 11 29 PM

### CODE REVIEW

File: javascript-forms/form-events.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+function handleInput(event) {
+  console.log(event.target.name);
+  console.log(event.target.value);
+}
+var username = document.querySelector('#user-name');
+var useremail = document.querySelector('#user-email');
+var message = document.querySelector('#user-message');
+username.addEventListener('focus', handleFocus);
+useremail.addEventListener('blur', handleBlur);
+message.addEventListener('input', handleInput);
```
Comment (codingcodymiller) [inline javascript-forms/form-events.js:18]: Screen Shot 2021-02-03 at 11 21 06 AM
You are missing some event listeners here. Every form control element should have an event listener for `blur`, `focus`, and `input`.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Nice work so far! One note:

File: javascript-forms/form-events.js (hunk 1 above)
Comment (Mohamed-Elmessiry) [reply javascript-forms/form-events.js]: Screen Shot 2021-02-03 at 6 47 07 PM

**Review by codingcodymiller — APPROVED** [review]
Nice work!! 🎉

**Pull request closed without merging**